Project_Starting_Point/Client/Client.cpp

```diff
@@ -21,39 +21,41 @@ int main()
 	ClientSocket = socket(AF_INET, SOCK_STREAM, IPPROTO_TCP);
 	SvrAddr.sin_family = AF_INET;
 	SvrAddr.sin_port = htons(27001);
-	SvrAddr.sin_addr.s_addr = inet_addr("127.0.0.1");
-	connect(ClientSocket, (struct sockaddr*)&SvrAddr, sizeof(SvrAddr));
+	SvrAddr.sin_addr.s_addr = inet_addr("127.0.0.1"); 
+	connect(ClientSocket, (struct sockaddr*)&SvrAddr, sizeof(SvrAddr)); // connecting client with server
 
-	uiSize = GetSize();
-	for (unsigned int l = 0; l < uiSize; l++)
+	uiSize = GetSize(); // calling in-home function which returns size O(n)
+	for (unsigned int l = 0; l < uiSize; l++)  //O(n)
 	{
 		string strInput;
-		ifstream ifs("DataFile.txt");
-		for (unsigned int iStart = 0; iStart < l; iStart++)
+		ifstream ifs("DataFile.txt"); // opens ifstream object on File 
+		for (unsigned int iStart = 0; iStart < l; iStart++) // O(n) // this is costly and time consuming as it really will make it O(n^2) efficient
 			getline(ifs, strInput);
 
-		getline(ifs, strInput);
-		if (l > 0)
+		getline(ifs, strInput); 
+		if (l > 0) // if it's not the first line there are numbers of data following
```

Comment:
I think here we should try to use tabular vocabulary. We could write something like "If we are past the header / column name row"